vm/proto/vm_runtime.proto

```diff
@@ -1,29 +1,21 @@
 syntax = "proto3";
-package vm_runtime;
+package vm;
 option go_package = "./proto";
 
-service VmRuntime {
-    rpc Create(CreateRequest) returns (CreateResponse);
-    rpc Execute(ExecuteRequest) returns (ExecuteResponse);
+service VM {
+    rpc Execute(Request) returns (Response);
 }
 
-message CreateRequest {
+message Request {
     uint64 projectID = 1;
-    string content = 2;
-    string expParam = 3;
+    bytes code = 2;
```

Comment:
What's the initiative of combining two requests together? the payload of `code` binary would be huge in the communication

Reply:
Although there are currently two separate interfaces, they always call create execute during execution. So I feel it might be better to combine them into one interface. Of course, this is just a suggestion; feel free to modify

Reply:
https://github.com/iotexproject/w3bstream/blob/develop/vm/vm.go#L24-L29